connect-location/src/main/java/com/ifttt/location/ConnectLocation.java

```diff
@@ -132,21 +137,21 @@ public void setLoggingEnabled(Boolean enabled) {
     /**
      * Given the connection id passed in during initialization, fetch the connection data, and check if it has an
      * enabled {@link UserFeature} that uses location.
-     *
+     * <p>
      * This method should be used in addition to the {@link #init(Context, ConnectionApiClient)} or
      * {@link #init(Context, UserTokenProvider)} method in the initialization process, in order to set up
      * ConnectLocation to account for users who have the connection already enabled but hasn't had it set up in your
      * app. This method fetches the latest connection data, checks if, for the given user (via {@link UserTokenProvider},
      * there is an enabled {@link UserFeature} that uses Location trigger. If that is true, it will set up the geofences
      * and call the {@link LocationStatusCallback#onRequestLocationPermission()}.
-     *
+     * <p>
      * If you have a connection that uses Location service, you should call this method in the first Activity that your
      * users use your app, so that you can prompt the location permission request as soon as possible.
      *
-     * @param context Context instance of the place where the permission check happens.
-     * @param connectionId Connection ID to be activated.
+     * @param context                Context instance of the place where the permission check happens.
+     * @param connectionId           Connection ID to be activated.
      * @param locationStatusCallback Nullable {@link LocationStatusCallback} instance, can be used to listen to geofence
-     * status changes and location permission requests.
+     *                               status changes and location permission requests.
      * @return a {@link PendingResult} instance if we need to fetch the connection data, or null if we can reuse the
      * cached data.
      */
```

Comment:
Couple of extra left spaces introduced above, is it intentional? Also an extra tag `<p>` was added without the closing tag

Reply:
Yeah I think these are the format that Android Studio automatically applies to javadocs.